penguin/web/routes.py

```
@@ -965,10 +972,10 @@ def _restore_reasoning_variant_override(
 
 
 def _resolve_include_reasoning(value: Optional[Any]) -> bool:
```

Ruff (0.15.11)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)